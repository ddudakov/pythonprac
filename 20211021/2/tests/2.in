:deg a b a**b
deg 2 10
:abs x abs(x)+1
abs -4
quit "{}:{}"
