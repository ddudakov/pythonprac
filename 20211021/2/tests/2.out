1024
5
3:5
